Lib/test/test_httplib.py

```diff
@@ -3,6 +3,8 @@
 import io
 import itertools
 import os
+import sys
```

Comment:
This isn't being used anywhere!?

Reply:
Yeah, good catch. I think it remained here while I was working on this.